vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -155,6 +162,17 @@ def parse_pod(pod)
         end
       end
 
+      createdby_txt = pod.metadata.annotations.try("kubernetes.io/created-by")
```

Comment:
Is `try` the right method?  When people see `try` they are generaly thinking of the [one from activesupport](http://api.rubyonrails.org/classes/Object.html#method-i-try), which is for calling a method when the receive might be nil.  If this was something added by kubeclient, I recommend changing kubeclient to use a different method name...maybe something like `fetch`